demo-app/js/app.js

```diff
@@ -42,6 +43,15 @@
     }
     listContainer.addEventListener('hidden-new-content', updateNewIndicator);
 
+    function openGaiaDialog(evt) {
+      var detail = evt.detail;
+      var li = list.getItemAt(detail.index);
+      var text = li.querySelector('h3').textContent;
+      dialog.textContent = text + ' item clicked!';
```

Comment:
Which is here BTW https://github.com/etiennesegonzac/dom-scheduler/wiki/ScheduledList#data-source-api